package.json

```diff
@@ -31,17 +31,18 @@
   },
   "ava": {
     "files": [
-      "dist/test/**/*.test.js"
-    ],
-    "sources": [
-      "src/*.ts",
-      "src/**/*.ts"
+      "!dist/test/**/*.test.js",
```

Comment:
I'm very confused by this update. Doesn't this say that it's only testing files that don't satisfy these globs? I would expect this to mean that no tests are run at all.